peerdb-api/endpoints/create-peer/kafka.mdx

```diff
@@ -0,0 +1,103 @@
+---
+title: "Create a Kafka peer"
+---
+You can create a Kafka peer using this endpoint. Here is the request and response format for creating a Kafka peer.
+
+<Info> This peer is only supported in [PeerDB OSS](https://github.com/PeerDB-io/peerdb) and [PeerDB Enterprise](https://github.com/PeerDB-io/peerdb-enterprise) </Info>
```

Comment:
I don't think this is required as PeerDB Cloud has a banner mentioning only CH support

Reply:
it wouldn't be clear if a user goes to the instance directly right?